constants.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2025 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+package otlptranslator
+
+// MetricType is a representation of metric types from OpenTelemetry.
+// Different types of Sums were introduced based on their metric temporalities.
+// For more details, see:
+// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#sums
+type MetricType string
+
+const (
+	// MetricTypeNonMonotonicCounter represents a counter that is not monotonically increasing, also known as delta counter.
+	MetricTypeNonMonotonicCounter MetricType = "non-monotonic-counter"
+	// MetricTypeMonotonicCounter represents a counter that is monotonically increasing, also known as cumulative counter.
+	MetricTypeMonotonicCounter MetricType = "monotonic-counter"
```

Comment:
@carrieedwards for the prometheus delta support, do we have standard names yet for the counters? It would probably be nice to make them the same in both projects.  (I could imagine that in prometheus, monotonic counters are just "counter" and delta counters are something else).

Reply:
The string is not really important actually... we could make this an `iota` and not deal with names 😛

Reply:
updated to iota!